Bugsnag/Assets/Bugsnag/Editor/SymbolUpload/BugsnagCLI.cs

```diff
@@ -0,0 +1,184 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using UnityEngine;
+
+namespace BugsnagUnity.Editor
+{
+    internal class BugsnagCLI
+    {
+        private const string DOWNLOADED_CLI_VERSION = "2.6.2";
+        private readonly string DOWNLOADED_CLI_PATH = Path.Combine(Application.dataPath, "../bugsnag/bin/bugsnag_cli");
+        private readonly string DOWNLOADED_CLI_URL = $"https://github.com/bugsnag/bugsnag-cli/releases/download/v{DOWNLOADED_CLI_VERSION}/";
+        private readonly string _cliExecutablePath;
+        public BugsnagCLI()
+        {
+            var config = BugsnagSettingsObject.GetSettingsObject();
+
+            if (string.IsNullOrEmpty(config.BugsnagCLIExecutablePath))
+            {
+                _cliExecutablePath = DownloadBugsnagCLI();
+            }
+            else
+            {
+                _cliExecutablePath = config.BugsnagCLIExecutablePath;
+            }
+            if (!File.Exists(_cliExecutablePath))
+            {
+                throw new Exception($"BugSnag CLI not found at path: {_cliExecutablePath}");
+            }
+        }
+
+        public void UploadAndroidSymbols(string buildOutputPath, string apiKey, string versionName, int versionCode)
+        {
+            string args = $"upload unity-android --api-key={apiKey} --verbose --project-root={Application.dataPath} {buildOutputPath}";
+
+            if (!string.IsNullOrEmpty(versionName))
+                args += $" --version-name={versionName}";
+            if (versionCode > -1)
+                args += $" --version-code={versionCode}";
+
+            int exitCode = StartProcess(_cliExecutablePath, args, out string output, out string error);
+
+            if (exitCode != 0)
+            {
+                throw new Exception($"Error uploading symbols: {error}\nOutput: {output}");
```

Comment:
How well does this render in the Unity Editor?

Is it worth returning stdout and stderr as outputs here as well, then the Uploader can output stdout as a separate (LogDebug?) in both the successful and unsuccessful case.